packages/target-tools/src/transforms/transformUtils.js

```diff
@@ -0,0 +1,267 @@
+    return ChannelTypeAEP.Browser;
+  }
+
+  if (targetChannel === ChannelType.Mobile) {
+    return ChannelTypeAEP.Browser;
+  }
+
+  return undefined;
+}
+
+export function createIdentityItem(id, authenticatedState, isPrimary) {
+  return {
+    id,
+    authenticatedState,
+    primary: isPrimary
+  };
+}
+
+export function toHours(minutesValue) {
+  if (!isNumber(minutesValue)) {
+    return undefined;
+  }
+
+  return minutesValue / MINUTES_PER_HOUR;
+}
+
+export function targetProductToAEP(product) {
+  if (isUndefined(product)) {
+    return undefined;
+  }
+
+  const { id, categoryId } = product;
+
+  return {
+    SKU: id,
+    category: categoryId
+  };
+}
+
+export function targetOrderToAEP(order) {
+  if (isUndefined(order)) {
+    return undefined;
+  }
+
+  const { id, total, purchasedProductIds } = order;
+
+  return {
+    order: {
+      purchaseId: id,
+      priceTotal: total
+    },
+    purchases: {
+      id:
+        isArray(purchasedProductIds) && purchasedProductIds.length > 0
+          ? purchasedProductIds[0]
+          : undefined
+    }
+  };
+}
+
+export function aepFormatToTargetOptionType(aepFormat) {
+  if (aepFormat === ContentTypeAEP.HTML) {
+    return OptionType.Html;
+  }
+
+  if (aepFormat === ContentTypeAEP.DOM_ACTION) {
+    return OptionType.Actions;
+  }
+
+  if (aepFormat === ContentTypeAEP.JSON) {
+    return OptionType.Json;
+  }
+
+  if (aepFormat === ContentTypeAEP.METRIC_ACTION) {
+    return OptionType.Dynamic;
+  }
+
+  if (aepFormat === ContentTypeAEP.URI_LIST) {
+    return OptionType.Redirect;
+  }
+
+  return undefined;
+}
+
+export function isGlobalMbox(mboxName) {
+  return mboxName === DEFAULT_GLOBAL_MBOX;
+}
+
+export const byIndex = (itemA, itemB) =>
+  (itemA.index || 0) - (itemB.index || 0);
+
+export function sanitize(object) {
+  return objectWithoutUndefinedValues(object, true);
+}
+
+export /**
+ *
+ * @param {string} key
+ * @param {array} list
+ * @returns {object}
+ */
+function indexBy(key, list, group = false) {
+  return list.reduce((indexed, item) => {
+    const keyValue = item[key];
+
+    if (group) {
+      if (isUndefined(indexed[keyValue])) {
+        // eslint-disable-next-line no-param-reassign
+        indexed[keyValue] = [];
+      }
+
+      indexed[keyValue].push(item);
+    } else {
+      // eslint-disable-next-line no-param-reassign
+      indexed[keyValue] = item;
+    }
+
+    return indexed;
+  }, {});
+}
+
+function createActionWithPropertyReplacements(propertyMapping, data) {
+  const action = {};
+
+  propertyMapping.forEach(property => {
+    const { aep, target } = property;
+    if (Object.prototype.hasOwnProperty.call(data, aep)) {
+      action[target] = data[aep];
+    }
+  });
+
+  return action;
+}
+
+export function createVecAction(data) {
+  return createActionWithPropertyReplacements(VEC_ACTION_PROPERTY_MAP, data);
+}
+
+export function isFormBasedActivity(data) {
+  const { selector } = data;
+  return selector === HEAD;
```

Comment:
we shouldn't determine the action as Form-based based on HEAD selector, as a HEAD selector can also be specified in a VEC action (by editing the selector in the UI)
I'd suggest differentiating FBC vs VEC only based on item schema